docs/deployment/teams.md

```diff
@@ -56,7 +56,7 @@ Members can't view or perform any actions in the following pages:
 
 ### Invite a team member through email
 
-Add team members easily to your organization by sending them an email. This email contains instructions for them to join your org through the same auth provider configured for your account.
+Add team members easily to your organization by sending them an email. This email contains instructions for them to join your org through the same auth provider configured for your account. The invitation does not grant access to users who are not provisioned in the configured auth provider.
```

Comment:
```suggestion
Add team members easily to your organization by sending them an email. This email contains instructions for them to join your org through the same auth provider configured for your account. The invitation only grants access to users who are already provisioned in the configured auth provider.
```

Feel free to ignore, but inverting this made it a bit less confusing for me (but this could just be a me problem)

Reply:
The invitation doesn't actually grant access at all, so the negative framing does have a different meaning here. But I'm open to something like "The invitation only facilitates access for users who are already", how does that sound?

Reply:
Yes, I like this wording!